### PR TITLE
XMC1400 XTREME Connectivity Kit

### PR DESCRIPTION
**Pull request by oscarcreativeshop**
Getting started documentation has been added along with wiki.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,6 @@
 [submodule "optiga-trust-m"]
 	path = optiga-trust-m
 	url = https://github.com/Infineon/optiga-trust-m
+[submodule "optiga-trust-m-1"]
```
Comment (ayushev) [inline .gitmodules]: i know you did this for the timebeing, so please move to the trust-m repo, not trust-m-1

File: README.md [hunk 2]
```diff
@@ -33,8 +37,30 @@ In this guide you may find the following steps:
     * Link to download manual: J-Link manual [Download](https://www.segger.com/downloads/jlink/)
 * Download and install [Tera Term](https://osdn.net/projects/ttssh2/releases/) (recommended)
 * Download this repository either via a direct [Download](https://github.com/Infineon/getstarted-optiga-trust-m/releases) or clone using `git clone --recursive`.
+## Quickstart XMC1400
```
Comment (ayushev) [inline README.md]: I don't see a need to have two almost identical descriptions. Please update the existing one to reflect both options.

File: README.md [hunk 3]
```diff
@@ -86,7 +112,7 @@ In this guide you may find the following steps:
 
 ## Compile and Debug
 
-The guidance on how to compile and debug the setup you can find [here](https://github.com/Infineon/getstarted-optiga-trust-m/wiki)
+The guidance on how to compile and debug the setup you can find [here](https://github.com/oscarcreativeshop/getstarted-optiga-trust-m/wiki)
```
Comment (ayushev) [inline README.md]: this should be infineon

**Review by ayushev — CHANGES_REQUESTED** [review]
+, please:
remove getstarted-optiga-trust-m.wiki submodule
use standard trust-m repo, not trust-m-1
rename xmc1400-getstarted-optiga-trust-m to "xmc1400_xtreme_kit" (the same manner as the "xmc4800_iot_kit")
Don't updte the xmc4800 project, it should be left untouched
No build directories shall be submitted (Release/Debug)
Only one build configuration (preferably "Debug")

**Review by ayushev — CHANGES_REQUESTED** [review]
Please remove .metadata folder
as before remove all DEBUG or RElease uild folders thay are not required for the build when you move the project

**Comment by oscarcreativeshop** [comment]
I missed this one,
Done
thanks!
Oscar

**Pull request merged**